style.css

```diff
@@ -3,108 +3,151 @@
   box-sizing: border-box;
  }
 
- body
- {
-   margin: 0;
-   padding: 0;
-   font-family: sans-serif;
-   background: #ffffff;
+ body{
+  background-color: #222222;
+  padding: 0;
+  margin: 0;
  }
 
+
 .header {
+  display: flex;
   padding-left: 1.875rem;
   text-align: left;
-  color: dodgerblue;
+  color: #ff6cc9;
+  border-bottom: thin solid black;
+  background: radial-gradient(circle at 10% 20%, rgb(10, 10, 10) 0%, rgb(34,34,34) 90.2%);
 }
 
-.footer {
-  height: 1.25rem;
+.main {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  /* background-color: #28282B; */
+ }
+
+.alt-wikipedia-container{
+  display: flex;
+  align-items: center;
 }
 
- .main {
-   display: flex;
-  justify-content: space-between;
- }
+.previous-button-container{
+  z-index: 1;
+  transform: translate(9.4rem, 0rem);
+}
 
- .wikipedia-content {
-     width: 31.25rem;
-     height: 31.25rem;
-     margin-top: 3.125rem;
- }
+.previous-wikipedia-content {
+    opacity: 0.2;
+    padding-right: 3.13rem;
+}
 
- .main-wikipedia-content {
+.next-button-container{
```

Comment:
Nit: your formatter should be hitting these files, too.

Reply:
I have added a formatter for the CSS file.